Tests/Adapter/DoctrineCacheAdapterTest.php

```diff
@@ -40,23 +43,51 @@ public function setUp()
         );
         $this->adapter = new DoctrineCacheAdapter($this->cacheProvider);
 
-        $this->cacheItem = $this->getMock('Psr\Cache\CacheItemInterface');
+        $this->cacheKey = 'cache-key';
```

Comment:
Though about it. I just do not like exposing unnecessary meta data. Thats just an implementation detail of a test class and is not mean to be used/seen anywhere outside it.